pkg/option/config.go

```diff
@@ -635,6 +638,10 @@ const (
 	// entries.
 	SockRevNatEntriesName = "bpf-sock-rev-map-max"
 
+	// EgressPolicyMapName configures max entries for egress gateway's policy
+	// map.
+	EgressPolicyMapName = "egress-gw-policy-max"
```

Comment:
```suggestion
	EgressGatewayPolicyEntriesName = "egress-gw-policy-map-max"
```

Reply:
Thanks for your reviewing! Done.

Reply:
You still have occurrences of `EgressPolicyMapName`, now undefined, causing the failures in the CI. Can you please update?